src/content/docs/ra-primer/relations.mdx

```diff
@@ -27,7 +27,7 @@ Consider the idea *is less than*. This is a *binary relation*. Binary because it
 | **4** | n | n | n | n | y |
 | **5** | n | n | n | n | n |
 
-One way of describing this is: *For every pair of numbers 1-5, pick out the ones where the first is smaller than the second.* If we write the pairs like this: *(x, y)*, the relation picks out this list:
+One way of describing this is: *For every pair of numbers 1-5, pick out the ones where the first is smaller than the second.* If we write the pairs like this: *(x, y)*, the relation picks out the list below, and conveys a clear semantics "*x* is less than *y*":
```

Comment:
I don't feel this really clarify anything, unless we start explaining what "semantics" means here (the semantics of semantics...). As it is, it just seems repetitive.